source/docs/software/vision-processing/identifying-and-processing-the-targets.rst

```diff
@@ -50,14 +50,14 @@ The Area score is calculated by comparing the area of the particle compared to t
 Aspect Ratio
 ^^^^^^^^^^^^
 
-The aspect ratio score is based on (Particle Width / Particle Height). The width and height of the particle are determined using something called the "equivalent rectangle". The equivalent rectangle is the rectangle with side lengths :math:`x` and :math:`y` where :math:`2x+2y` equals the particle perimeter and :math:`x \cdot y` equals the particle area. The equivalent rectangle is used for the aspect ratio calculation as it is less affected by skewing of the rectangle than using the bounding box.  When using the bounding box rectangle for aspect ratio, as the rectangle is skewed the height increases and the width decreases.
+The aspect ratio score is based on (Particle Width / Particle Height). The width and height of the particle are determined using something called the "equivalent rectangle". The equivalent rectangle is the rectangle with side lengths :math:`x` and :math:`y` where :math:`2x+2y` equals the particle perimeter and :math:`x \cdot y` equals the particle area. The equivalent rectangle is used for the aspect ratio calculation as it is less affected by skewing of the rectangle than using the bounding box. When using the bounding box rectangle for aspect ratio, as the rectangle is skewed the height increases and the width decreases.
 
 The target is 20" wide by 12" tall, for a ratio of 1.6. The detected aspect ratio is compared to this ideal ratio. The aspect ratio score is normalized to return 100 when the ratio matches the target ratio and drops linearly as the ratio varies below or above.
 
 Moment
 ^^^^^^
 
-The moment measurement calculates the particles moment of inertia about it's center of mass. This measurement provides a representation of the pixel distribution in the particle. The ideal score for this test is ~0.28. See: `Moment of Inertia <https://en.wikipedia.org/wiki/Moment_of_inertia>`_
+The moment measurement calculates the particles moment of inertia about it's center of mass. This measurement provides a representation of the pixel distribution in the particle. The ideal score for this test is ~0.28. See: `Moment of Inertia <https://en.wikipedia.org/wiki/Moment_of_inertia>`__
```

Comment:
Only one underscore as per http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets
```suggestion
The moment measurement calculates the particles moment of inertia about it's center of mass. This measurement provides a representation of the pixel distribution in the particle. The ideal score for this test is ~0.28. See: `Moment of Inertia <https://en.wikipedia.org/wiki/Moment_of_inertia>`_
```

Reply:
Again, the style guid says otherwise.  Sounds link we should fix the style guide.